common/client.go

```diff
@@ -411,19 +411,39 @@ func (c *DatabricksClient) configureHTTPCLient() {
 	}
 }
 
+func (c *DatabricksClient) GetHost() string {
+	if c.Host == "" {
+		err := c.Authenticate(context.TODO())
+		if err != nil {
+			panic(err)
```

Comment:
No panics in production flow. It'll crash provider and customers would loose faith again

Reply:
so just return empty string?

Reply:
fixed